src/wp-includes/global-styles-and-settings.php

```diff
@@ -374,6 +364,45 @@ function ( $item ) {
 	}
 }
 
+/**
+ * Gets the block name from a given theme.json path.
+ *
+ * @since 6.3.0
+ * @access private
+ *
+ * @param string $path Path to a property in theme.json.
+ * @return string Identified block name, or empty string if none found.
+ */
+function wp_get_block_name_from_theme_json_path( $path ) {
+	// Block name is expected to be the third item after 'styles' and 'blocks'.
+	if (
+		count( $path ) >= 3
```

Comment:
Path is a string isn't it? It is treated like an array here.

Reply:
@spacedmonkey It is an array actually (similar to what would be used with e.g. `_wp_array_get()`), so this logic is correct. I realized because of your comment though that I had incorrectly documented the parameter, so I fixed that.